### PR TITLE
Describe how to test changes in Plone and handle translations

### PR DESCRIPTION
**Pull request by vincentfretin**

### CODE REVIEW

**Comment by vincentfretin** [comment]
@frapell @djowett-ftw did you know all this? Or I'm the only one that had a hard time figuring all this out? TT
No wonder why nobody fixed https://github.com/plone/Products.CMFPlone/issues/2188 since 2017! :-p
Are the instructions understandable? Can you reproduce it? :-)

**Comment by djowett-ftw** [comment]
> @frapell @djowett-ftw did you know all this? Or I'm the only one that had a hard time figuring all this out? TT

No, I didn't - and I think you got further than me on it (though don't worry I still had a hard time).  I'm trying to get some time to work on this today or tomorrow.   Are there Plone 5 releases imminent?

**Comment by vincentfretin** [comment]
Yes, it should be this week I think. https://github.com/plone/Products.CMFPlone/issues/3119

**Comment by frapell** [comment]
@vincentfretin Yes, unfortunately documentation is not complete on this, so thank you for expanding it...

In here you can find the training we gave some years ago with @thet about working with mockup https://training.plone.org/5/javascript/index.html have you seen it?

There's also this https://docs.plone.org/adapt-and-extend/theming/resourceregistry.html

There's nothing about translations though....

**Comment by vincentfretin** [comment]
Thanks for the links @frapell . I may have read some of it a few years ago, but I didn't do any Plone 5 development for a few years now, so I may have forgot some things. :-) I'll read that again and see if I learn new things.

File: README.rst [hunk 1]
```diff
@@ -42,10 +42,45 @@ Run tests with Chrome::
...
+    $ cd ../..
+    $ cp src/mockup/widgets.pot src/plone.app.locales/plone/app/locales/locales/widgets.pot
+    $ bin/buildout -c experimental/i18n.cfg  # to have the bin/i18n command
+    $ bin/i18n widgets
+
+If you did some changes in the js files and want to test them live in Plone:
+
+- go to Site Setup -> Resource Registries
+- click on the "Development Mode (only logged in users)" checkbox
+- click on the "Develop Javascript" button for the plone-editor-tools bundle
```
Comment (djowett-ftw) [inline README.rst:63]: plone-editor-tools isn't in Plone 5.1, "plone-logged-in" would be the equivalent there IIUC

Comment (vincentfretin) [reply README.rst:63]: You're probably right. This PR is for mockup master so targeting Plone 5.2.x.

**Pull request merged**